scripts/snackbar.js

```diff
@@ -0,0 +1,10 @@
+["1", "2", "3"].forEach((_id) => {
+  const openBtn_1 = document.getElementById("kash-snackbar-open-btn-" + _id);
+  const _snackbar = document.getElementById("kash-snackbar-" + _id);
+  openBtn_1.onclick = () => {
+    _snackbar.style.display =
+      _snackbar.style.display === "none" ? "flex" : "none";
+  };
+  const closeBtn = document.getElementById("kash-snackbar-close-btn-" + _id);
+  closeBtn.onclick = () => (_snackbar.style.display = "none");
```

Comment:
Why is _snackbar.style.display="none"  in ()?

Reply:
I guess prettier did that previously i wrote one more line which i removed later